charts/pulsar/templates/proxy-statefulset.yaml

```diff
@@ -300,7 +300,7 @@ spec:
         - name: ca
           secret:
             {{- if eq .Values.certs.internal_issuer.type "selfsigning" }}
```

Comment:
It looks like `secretName: "{{ template "pulsar.certs.issuers.ca.secretName" . }}"` should be used also here for consistency.

Reply:
pushed a commit to address this.